breakmodel.py

```diff
@@ -277,7 +284,9 @@ def dispatch_model_ex(
             called directly during the forward, for instance if a `dense` linear layer is registered, but at forward,
             `dense.weight` and `dense.bias` are used in some operations instead of calling `dense` directly.
     """
-    if main_device != "cpu":
+    if utils.args.use_ipex:
+        pass
```

Comment:
Unused?

Reply:
Yes. A copy paste from the older fork. Will remove.